### ISSUE TITLE
Add params to toggle verbose/non verbose mode

### ISSUE THREAD

**Issue by MangelMaxime** [Zaid-Ajaj/Femto#2]
I find the current logs really helpful to identify what is being done. However, if you have a lot of dependencies, it can quickly become hard to read because of the noise.

I think we could just log the dependencies that are failing and have a summary at the end.

> 
> Fulma depends on npm package 'bulma':
> ...
> ...
> 
> Femto 10 dependencies checked for 9 passed, 1 failed. Failed.

And when in verbose mode, we would have all the actions like done today.

What do you think?

**Comment by Zaid-Ajaj** [comment]
I agree that a compact log would be good but in the first stages I would like to keep the verbose mode as it is now because it will help identity bugs and corner cases fast, the non verbose log mode can be added later
